data/pathfinder/paizo/player_companion/adventurers_armory/aa_feats.lst

```diff
@@ -9,7 +9,7 @@ SOURCELONG:Adventurer's Armory	SOURCESHORT:AA	SOURCEWEB:http://paizo.com/product
 # Ability Name			Category of Ability	Type			Required Ability					Description																																																																							Source, Long Desc.			Source Page		Benefits
 Rapid Reload (Spear-thrower)	CATEGORY:FEAT		TYPE:Combat									DESC:You can reload your spear-thrower quickly.																																																																							SOURCEPAGE:p.8	BENEFIT:You can reload a spear-thrower as a free action.
 
-#TODO Equipment Trick p.22
+Equipment Trick				CATEGORY:FEAT				TYPE:Combat						PRETOTALAB:1																																					PRETEXT:Base attack bonus +1.																			DESC:You understand how to use a certain item in combat.							SOURCEPAGE:p.22			BENEFIT:You may use any equipment tricks relating to the item if you meet the appropriate trick requirements. If the item would normally be considered an improvised weapon, you may treat it as a normal weapon or an improvised weapon, whichever is more beneficial for you. You can gain the Equipment Trick feat multiple times. Each time you take the feat, it applies to a new type of equipment.		STACK:NO	MULT:YES	CHOOSE:ABILITY|Special Ability|TYPE=EquipmentTrick
```

Comment:
Yes. There are several different pieces of equipment, so the player must pick one for the trick.